internal/rss.go

```diff
@@ -25,7 +28,15 @@ func (r *RSS) Init(c *Controller) {
 
 	// Check if we have any OMPL file to load
 	if r.c.conf.OPMLFile != "" {
-		doc, err := opml.NewOPMLFromFile(r.c.conf.OPMLFile)
+		opmlpath := r.c.conf.OPMLFile
+		// if the path contains ~
+		if strings.HasPrefix(opmlpath, "~/") {
+			usr, _ := user.Current()
```

Comment:
Should probably check for errors here. Then print an appropriate message in case it fails and return.